src/esp/metadata/managers/StageAttributesManager.cpp

```diff
@@ -202,12 +202,20 @@ StageAttributes::ptr StageAttributesManager::initNewObjectInternal(
       newAttributes->setNavmeshAssetHandle(navmeshFilename);
     }
     // Build default semantic descriptor file name
-    std::string houseFilename = io::changeExtension(attributesHandle, ".house");
+    std::string houseFilename = "";
+    for (const auto& ext : {".house", ".scn"}) {
+      const std::string proposed = io::changeExtension(attributesHandle, ext);
+      if (Cr::Utility::Directory::exists(proposed)) {
+        houseFilename = proposed;
+        break;
+      }
+    }
 
     if (!Corrade::Utility::Directory::exists(houseFilename)) {
       houseFilename = io::changeExtension(attributesHandle, ".scn");
     }
```

Comment:
This should probably be removed - it seems to be covered by the newly added code.

Reply:
Whoops, I didn't think that ended up in this PR.